rudder-agent/SPECS/dependencies

```diff
@@ -1,9 +1,9 @@
 sles-10sp3:gcc bison flex autoconf automake libtool pcre-devel zlib-devel pam-devel libxml2-devel curl-devel libacl-devel
 sles-11:gcc openssl-devel bison flex autoconf automake libtool pcre-devel zlib-devel libbz2-devel pam-devel libxml2-devel libcurl-devel libacl-devel
 sles-12:gcc openssl-devel bison flex autoconf automake libtool pcre-devel zlib-devel libbz2-devel pam-devel libyaml-devel libxml2-devel curl-devel libacl-devel
-sles-15:gcc openssl-devel bison flex autoconf automake libtool pcre-devel zlib-devel libbz2-devel pam-devel libyaml-devel libxml2-devel curl-devel libacl-devel
+sles-15:gcc openssl-devel bison flex autoconf automake libtool jq pcre-devel zlib-devel libbz2-devel pam-devel libyaml-devel libxml2-devel curl-devel libacl-devel
```

Comment:
File is used to automatically install packager on the builder, so it is a build dependency.
So you need the -devel version of the package